packages/lwc/README.md

```diff
@@ -0,0 +1,73 @@
+Lightning Web Components (LWC) is a blazing fast, enterprise-grade web components foundation for building user interfaces. LWC provides a simple authoring format for UI Components. LWC compiles this authoring format into low-level Web Component APIs. The `lwc` package is the main entry point for dependencies.
```

Comment:
Re blazing fast, we provide no data justifying this claim.

Reply:
Let's focus on enterprise-grade only. I think we drop blazing-fast unless we quantify.

Reply:
+1 we did that when we launch to differenciate from aura mostly. I think now   the message is not needed